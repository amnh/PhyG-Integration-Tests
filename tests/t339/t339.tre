(AlbanianTosk:3900.05,(KolschColognian:6186.4,(AncientIonicGreek:3471.3,(SanskritVedic:3318.6,(EasternFrisianSaterland:2403.1,(Danish:2414.95,(ArmenianWestern:2831.45,((Manx:1329.15,(Cornish:1487.6,(Faroese:1776.1999999999998,Yiddish:4099.65)HTU26:1245.7)HTU25:423.1)HTU24:1594.1,((Irish:1774.45,Scots:2108.8)HTU28:1679.1,(Assamese:5870.200000000001,(Icelandic:6501.45,OldPrussian:1677.65)HTU30:2352.1)HTU29:1450.6499999999999)HTU27:1205.6)HTU23:899.1)HTU22:897.65)HTU21:1139.1)HTU20:2292.4)HTU19:1421.6)HTU18:1195.95)HTU17:3663.55)HTU16[48504.4];
(AlbanianTosk:4363.55,(KolschColognian:6197.4,(AncientIonicGreek:3471.3,(SanskritVedic:3336.6,(EasternFrisianSaterland:2383.6,(Danish:2378.45,(ArmenianWestern:2784.45,((Manx:1289.15,(Cornish:1667.6,Faroese:2057.8)HTU25:492.1)HTU24:1587.6,((Irish:1750.35,Scots:2032.9)HTU27:1707.6,((Assamese:4828.6,Yiddish:5268.1)HTU29:2554.2999999999997,(Icelandic:6172.55,OldPrussian:2173.05)HTU30:2630.6)HTU28:1880.1499999999999)HTU26:1336.0)HTU23:1022.6)HTU22:949.6500000000001)HTU21:1190.6)HTU20:2304.9)HTU19:1432.1)HTU18:1175.95)HTU17:3905.55)HTU16[47887.3];
(AlbanianTosk:4337.55,(KolschColognian:6208.4,(AncientIonicGreek:3478.3,(SanskritVedic:3355.1,(EasternFrisianSaterland:2424.1,(Danish:2357.45,(ArmenianWestern:2888.45,((Manx:1403.3000000000002,(Cornish:1610.6,(Faroese:1129.1,Scots:1558.1)HTU26:1344.3000000000002)HTU25:509.95)HTU24:1485.1,(Irish:2758.1,((Assamese:4721.1,Yiddish:5133.6)HTU29:2308.3,(Icelandic:6261.55,OldPrussian:2197.55)HTU30:2461.1)HTU28:1712.4)HTU27:1162.1)HTU23:928.6)HTU22:991.8000000000001)HTU21:1216.6)HTU20:2287.4)HTU19:1405.1)HTU18:1193.95)HTU17:3838.55)HTU16[47350.2];
(AlbanianTosk:2446.55,(KolschColognian:6388.4,((AncientIonicGreek:3118.3,SanskritVedic:2834.6)HTU17:3514.6,((Icelandic:5611.05,OldPrussian:2788.55)HTU22:2343.6,((Assamese:3603.6,Yiddish:3557.6)HTU21:2049.3,((Irish:1885.6,Scots:2046.1499999999999)HTU23:885.1,((Manx:1514.3000000000002,(Cornish:1830.6,Faroese:2104.8)HTU25:589.45)HTU26:846.6,(ArmenianWestern:2190.95,(Danish:1849.95,EasternFrisianSaterland:2025.6)HTU30:422.29999999999995)HTU29:559.1)HTU28:880.6)HTU27:1851.4)HTU24:1352.6)HTU19:1874.1)HTU20:3107.95)HTU18:1382.55)HTU16[46374.4];
(AlbanianTosk:2516.05,(KolschColognian:6421.9,((AncientIonicGreek:3143.3,SanskritVedic:2794.6)HTU17:3116.1,((Icelandic:5736.55,OldPrussian:2592.05)HTU22:1903.6,((Assamese:3675.1,Yiddish:3500.6)HTU21:1702.3000000000002,(Irish:2079.6,((Manx:1422.8000000000002,(Cornish:1638.6,(Faroese:1152.6,Scots:1587.1)HTU24:1364.3000000000002)HTU25:525.95)HTU26:710.1,(ArmenianWestern:2278.45,(Danish:1709.45,EasternFrisianSaterland:2111.6)HTU30:529.8)HTU29:597.1)HTU28:830.1)HTU27:1868.9)HTU23:1294.1)HTU19:1800.1)HTU20:2691.45)HTU18:1452.05)HTU16[45933.3];
(AlbanianTosk:2751.05,(KolschColognian:6523.9,((AncientIonicGreek:3152.8,SanskritVedic:2786.1)HTU23:3081.6,((Icelandic:5724.05,OldPrussian:2749.05)HTU21:1841.6,((Assamese:3444.1,Yiddish:3505.1)HTU19:1589.8000000000002,(Irish:2167.1,((Danish:1794.95,EasternFrisianSaterland:1937.6)HTU17:676.3,(ArmenianWestern:2277.95,((Faroese:1155.1,Scots:1538.6)HTU29:760.3,(Cornish:1450.1,Manx:1140.65)HTU30:1071.6)HTU27:481.1)HTU28:521.45)HTU18:765.1)HTU20:1730.9)HTU22:1153.1)HTU24:1632.1)HTU25:2663.95)HTU26:1504.05)HTU16[45131.7];
