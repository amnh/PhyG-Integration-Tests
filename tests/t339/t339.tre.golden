(AlbanianTosk:2409.0,(KolschColognian:5274.8,(AncientIonicGreek:2447.2,(SanskritVedic:1974.0,(EasternFrisianSaterland:982.0,(Danish:1017.9,(ArmenianWestern:1543.0,((Manx:341.2,(Cornish:670.0,(Faroese:510.0,Scots:933.0)HTU26:975.7)HTU25:77.4)HTU24:199.0,(Irish:768.5,((Assamese:2667.5,Yiddish:2791.0)HTU29:476.2,(Icelandic:5178.0,OldPrussian:1016.0)HTU30:310.0)HTU28:139.9)HTU27:96.0)HTU23:52.0)HTU22:178.2)HTU21:383.0)HTU20:1135.9)HTU19:270.0)HTU18:490.9)HTU17:1557.0)HTU16[47350.2];
